querybook/webapp/components/BoardDetailedList/BoardDetailedList.tsx

```diff
@@ -30,54 +29,57 @@ const BoardListItem: React.FunctionComponent<{
     } = board;
     const { userInfo: ownerInfo, loading } = useUser({ uid: ownerUid });
 
-    const richTextDescription = useMemo(
```

Comment:
Added this because charlie suggested memoizing the rich text description in my PR, curious why you removed it?